redisinsight/desktop/forge.config.js

```diff
@@ -0,0 +1,61 @@
+const { FusesPlugin } = require('@electron-forge/plugin-fuses');
```

Comment:
Is this file used? I can't find a link to it anywhere.

Reply:
You are right. I've missed to clean it up after when I started removing all different experiments. Removed.